assets/src/infrastructure/ui/display/Alert/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { IAlert } from '@chakra-ui/core';
+
+import Error from './Error';
+
+interface Props extends IAlert {
+	description: string;
+	title: string;
+}
```

Comment:
Lets use a specific name for props for all the components.

```suggestion
interface AlertProps extends IAlert {
	description: string;
	title: string;
}
```

Reply:
agree, I was `Props` on purpose, since it wasn't exported;
but it's better to have consistency; updated